rails-time-track/spec/controllers/api/projects_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Api::ProjectsController, type: :controller do
+
+  before do
+    @projects = Project.create(
+      name: "Project Rumi",
+      client: "MINEDU",
+      category: "Eduaction",
+      closed: false
+    )
+  end
```

Comment:
Hi Paulo, in this case, if you want to be sure that only the "open" projects are being returned, you need to create a fake data that includes at least 2 projects (one open and one closed). What do you think?

Reply:
I think that it's a good idea because it's necessary to be sure.